### ISSUE TITLE
Add catalog/report choice to filename?

### ISSUE THREAD

**Issue by dmundra** [GSA/openacr#343]
From https://github.com/GSA/openacr-editor/pull/55

> The catalog type should be appended to the file name.
> * 508
> * WCAG
> * WCAG+508
> 
> Probably an enhancement

The catalog is captured in the YAML but could be added to the file name if needed.
